src/IssueViz/Editor/TaggableBufferIndicator.cs

```diff
@@ -54,7 +56,16 @@ public bool IsTaggable(ITextBuffer buffer)
                 return false;
             }
 
-            return fileSystem.File.Exists(filePath);
+            var exists = fileSystem.File.Exists(filePath);
+
+            if (!exists)
+            {
+                return false;
+            }
+
+            var isUnderTemp = PathHelper.IsPathRootedUnderRoot(filePath, Path.GetTempPath());
```

Comment:
No reason in particular; since `System.IO.Path.GetTempPath` uses WinAPI to get the running machine's temp folder, it doesn't leave any junk or require anything to exist. I could mock it but `PathHelper.IsPathRootedUnderRoot` uses `System.IO.Path` anyway, so I didn't see the point. We would need an interface for `PathHelper.IsPathRootedUnderRoot` (which we should have since it has some logic in it), but it's out of scope for this task so I left it as is.

Reply:
Okay, sounds reasonable!